### ISSUE TITLE
Weekly Scrum - Nov 17

### ISSUE THREAD

**Issue by Zhangjason308** [Zhangjason308/SYSC-4806-Project#66]
Weekly Updates for Week 1 of Milestone 2

**Comment by Zhangjason308** [comment]
Jason

**This Week**
- I worked on adding the Server classes/entity for the User class, as well as adding the the controller class for the API requests to be used in the frontend (Issue #38 )
- Started working on the Login page in svelte

**Next Week**
- Work on the Register svelte and profile svelte
- Add more requests for the profile and register logic
- Make sure the routing works
- Add test cases

**Blockers**
- No blockers

**Comment by nicholasyung1** [comment]
Nick

This Week:
I started adding the Add new perk form screen. I finished the PerkController which is going to be merged into perkManagerController after being merged.

Next Week:
Complete Add new perk form screen and complete test cases.

Blockers
Other assignments

**Comment by Waluiyeegi** [comment]
Caleb 

This Week

- I started worked on displaying the perk which includes creating the svelt file.
- I also started working on the home page to make sure that the perks can be viewed properly.

Next Week

- Finish the work on displaying the perk and making sure that it is displayed properly on the home page
- Finish integration tests for Perk manager controller and unit tests for Perk manager.
- work on readme
- Start working on Delete Perk functionality

Blockers

- Other Assignments
